trojsten/polls/templates/trojsten/polls/view_question.html

```diff
@@ -0,0 +1,52 @@
+{% extends "trojsten/layout/main.html" %}
+{% load polls_parts %}
+
+{% block title %}Anketa{% endblock %}
+
+{% block page_header %}
+
+<h1> Anketa </h1>
+
+{% endblock %}
+
+{% block page_content %}
+
+{% if not questions %}
+
+    Neexistujú žiadne ankety.
+
+{% else %}
+
+    <h1> {{ current.text }} </h1>
+
+    {% show_time current %}
+
+    <form method="POST">
+    {% csrf_token %}
+    
+    <table>
+    {% for voteanswer in answers %}
+        <tr>
+            <td margin="10px" style="padding:0 10px 0 10px">{% if voteanswer.2 == user_vote %}<b>{% endif %} {{ voteanswer.0 }} {% if voteanswer.2 == user_vote %}</b>{% endif %}</td>
+            <td margin="10px" style="padding:0 10px 0 10px">{% if voteanswer.2 == user_vote %}<b>{% endif %} {{ voteanswer.1 }} {% if voteanswer.2 == user_vote %}</b>{% endif %}</td>
+            {% if not current.expired and request.user.is_authenticated %}
+            <td style="padding:0 10px 0 10px"><button class="btn btn-primary" type="submit" name="action" value="vote{{ voteanswer.2 }}">Hlasuj </button></td>

```

Comment:
_vote_ prefix netreba, zbytocne to komplikuje kod bez toho aby to pridalo hodnotu.